integration/testdata/debug/go/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM golang:1.12 as builder
+
+COPY app.go .
+# Must use eval to handle GOGCFLAGS with spaces like `-gcflags='all=-N -l'`
+ARG GOGCFLAGS
+RUN eval go build "${GOGCFLAGS}" -o /app .
+
+FROM gcr.io/distroless/base
+# use GOTRACEBACK=all to trigger `skaffold debug`
```

Comment:
not sure I understand this comment... can you clarify why GOTRACEBACK triggers `skaffold debug` ?

Reply:
Part of the language-detection heuristics.  Will add a comment.